### PR TITLE
Update the Terms and Conditions for the Scrum document

### PR DESCRIPTION
**Pull request by codificat**
## Related Issues and Dependencies

Fixes: https://github.com/open-services-group/community/issues/143

## This Pull Request implements

Update the Terms&Conditions for the Scrum document:

### Description

Changes include:

 - More structure/sections in the document.
 - Dedicated section for issue quality
 - Add references to the SIGs
 - Add effort estimates
 - Switch from 2-week to 3-week sprints

Also updating the roadmap document to fix links to projects.

### CODE REVIEW

**Comment by codificat** [comment]
I believe this is ready for review now.

As per https://github.com/open-services-group/community/issues/143, going to
/assign @Gkrumbach07 
for approval, but I believe this should be reviewed also by the team
/cc @goern @fridex

File: docs/TermsAndConditionsForTheScrum.md [hunk 1]
```diff
@@ -1,55 +1,168 @@
...
+
+Issues can belong to one of Thoth's [Special Interest Groups (SIGs)](../community/sig-list.md). These should have the corresponding `sig/` label and be tracked in the corresponding SIG's project.
+
+# Issue triage
+
+Everyone is encouraged to assist in making sure every issue meets good quality standards, starting with the person who creates the issue.
+
+The process of issue triaging involves making sure that the issues have enough quality.
+
+Ultimately, SIGs are resposible of ensuring that the issues relevant to their SIGs have an adequate quality.
```
Comment (Gkrumbach07) [inline docs/TermsAndConditionsForTheScrum.md:30]:
```suggestion
Ultimately, SIGs are responsible for ensuring that the issues relevant to their SIGs have an adequate quality.
```

**Review by Gkrumbach07 — CHANGES_REQUESTED** [review]
All looks good to me. See spelling suggestions

File: docs/TermsAndConditionsForTheScrum.md [hunk 2]
```diff
@@ -1,55 +1,168 @@
...
+<a name="requirements"></a>
+## Requirements for issues to be planned
+
+An issue must meet the following criteria before it can be planned:
+
+- have an assignee
+- be `triage/accepted`
+- have a defined priority that is aligned with the sprint goals
+
+Feature requests (`type/feature`) must also have:
```
Comment (goern) [inline docs/TermsAndConditionsForTheScrum.md:114]: type or kind?

Comment (codificat) [reply docs/TermsAndConditionsForTheScrum.md:114]: good catch, thanks. It is `kind/feature` indeed

**Pull request merged**